mycroft/skills/core.py

```diff
@@ -60,12 +60,28 @@ def load_vocab_from_file(path, vocab_type, emitter):
                                       'alias_of': entity}))
 
 
+def load_regex_from_file(path, emitter):
+    with open(path, 'r') as reg_file:
+        for line in reg_file.readlines():
+            re.compile(line.strip())
+            emitter.emit(
+                Message("register_vocab",
+                        metadata={'regex': line.strip()}))
+
+
 def load_vocabulary(basedir, emitter):
     for vocab_type in os.listdir(basedir):
         load_vocab_from_file(
             join(basedir, vocab_type), splitext(vocab_type)[0], emitter)
 
 
+def load_regex(basedir, emitter):
+    for regex_type in os.listdir(basedir):
+        if regex_type.endswith(".rx"):
```

Comment:
@jdorleans I don't think so. With this check, I'm checking each file in the directory to see if it has a `.rx` extension. If it does, it gets sent to to the `load_regex_from_file` method. 

If the check was in that method, it would result in more `load_regex_from_file` calls with no real benefit.